config/ergw-capwap-node.config

```diff
@@ -44,8 +44,16 @@
 		      ]
 		     }
 		    ]}
-		  ]}
-	  ]},
+		  ]},
+	  {location_provider, #{
+	    providers => [
+	        {capwap_loc_provider_default, #{default_loc => {location, Lat = <<"123">>, Long = <<"456">>}}},
+	        {capwap_loc_provider_http, #{uri => "http://127.0.0.1:9999", timeout => 30000}}
+            ],
+            refresh => RefreshTime}
```

Comment:
Ouch, copy/paste stuff. It doesn't crash currently because it's used only as example for the helm chart (contained in some other project). Changed, and thanks.